platforms/android/com/crittercism/plugin/CDVCrittercism.java

```diff
@@ -98,6 +104,14 @@ public boolean execute(String action, final JSONArray args,
         }
     }
 
+    private boolean executeInit(String action, final JSONArray args,
+                               CallbackContext callbackContext) throws JSONException {
+        final String appID = args.getString(0);
```

Comment:
I tested calling Crittercism.init with an empty argument on an Android device. Turns out PhoneGap (or javascript?) passed in 'null' string if user passes no arguments. We don't need a check here.